pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
correct, not the image ENTRYPOINT, but we have our own implicit entrypoint of "bash -c" that we want.

if we don't have that implicit entrypoint, 95% of these commands will need to look like:  ["bash","-c","rake","test"] instead of just ["rake","test"]